CHANGES.md

```diff
@@ -1,5 +1,27 @@
 # Changes
 
+## edge-21.2.2
+
```

Comment:
Most other edge releases have a short blurb here summarizing the most important changes at a high level.

Reply:
I think a lot of the recent one have had one because they've introduced some significant changes with all the extension work, but this edge did not feel that way to me.

I can include something about the `repair` command? All the other bullets felt pretty independent and there was no major behavior change.

Reply:
I think we should call out TCP multicluster support up top. It's a big feature!

Reply:
I kept it out because the service mirror change hasn't merged. I mentioned at the bottom though that we are preparing to support it.

Reply:
Separate from opaque ports the TCP support is there so I see what you're saying. I can include that.